modules/oadp-self-service-nab-nar-logs.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-non-admin-use-cases.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-self-service-nab-nar-logs_{context}"]
+= Reviewing NAB and NAR logs
+
+As a non-admin user, you can review the logs for the NAB and NAR CRs by creating a `NonAdminDownloadRequest` (NADR) CR.
```

Comment:
I would like to the nonAdminDownloadRequest to show an example of pull the equivilant of the velero describe w/ details.. this is the most clear output we have for customers.
@kaovilai can you provide a clear example of a nonadmindownloadrequest that provides the user w/ describe --details  . I don't think we ever added a good example to https://github.com/migtools/oadp-non-admin/blob/8a0600cf9dc4944cf1aa3886e61010d0d92d3822/docs/draft_production_docs/oadp-self-service.md

Reply:
added [here](https://github.com/migtools/oadp-non-admin/pull/234#pullrequestreview-2774802022)